client/app/src/Command/ProcessOrgCSVCommand.php

```diff
@@ -65,32 +65,42 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
```

Comment:
If we have the command on the api side now, is this command still required on the client side?